src/WorkerDecorationV1Decorators/CrashedThresholdV1/CrashedThresholdDecorator.service.yml

```diff
@@ -0,0 +1,11 @@
+parameters:
+  Neighborhoods\KojoWorkerDecoratorComponent\WorkerDecorationV1Decorators\CrashedThresholdV1\CrashedThresholdDecoratorInterface.threshold: 5
+  Neighborhoods\KojoWorkerDecoratorComponent\WorkerDecorationV1Decorators\CrashedThresholdV1\CrashedThresholdDecoratorInterface.delaySeconds: 1
```

Comment:
Not really a blocker as much as a general question. Maybe this relates to a previous comment I made somewhere (though I can't seem to find it anymore, and which I might have not had enough context at the time), but how do we override these in userspace? And do they end up all being the same for every `CrashedThresholdDecorator`, or is there a way to specify them independently per stack? (e.g. I want tone job type to delay by an hour, but I want another to delay by only 5 minutes)

Reply:
Ah, here was the comment that I think I'm referring to: https://github.com/neighborhoods/KojoWorkerDecoratorComponentFitness/pull/1#issuecomment-824735474

Reply:
When the container builder encounters multiple definitions of the same service or parameter the later will be used. An example is given in the [fitness](https://github.com/neighborhoods/KojoWorkerDecoratorComponentFitness/tree/main/src/DecoratorParameters/Worker). It is necessary to include the vendor path before including the product path.

This works nicely for subset containers, but in integration tests the same parameter gets overridden multiple times.

Reply:
Gotcha, so in the case of an individual worker wanting separate parameters from others using the same decorator, we'd just need a different file per worker, and then the container builder would just need to know where to look?

Reply:
Yes